src/MgvOfferTakingWithPermit.sol

```diff
@@ -68,7 +68,7 @@ abstract contract MgvOfferTakingWithPermit is MgvOfferTaking {
 
   /* The delegate version of `marketOrder` is `marketOrderFor`, which takes a `taker` address as additional argument. Penalties incurred by failed offers will still be sent to `msg.sender`, but exchanged amounts will be transferred from and to the `taker`. If the `msg.sender`'s allowance for the given `outbound_tkn`,`inbound_tkn` and `taker` are strictly less than the total amount eventually spent by `taker`, the call will fail. */
```

Comment:
(why can't I comment on non-changed lines far away...)
comment for `deductSenderAllowance` states: "its both checks that"
changes "its" to "it"